src/main/java/frc/robot/Robot.java

```diff
@@ -154,27 +161,31 @@ public void teleopPeriodic() {
       ballChucker9000.moveIndexer(0);
     }
 
-    if (manipulator.getButton(Button.X)) {
-      climb.runWinch(0.5);
-    } else if (manipulator.getButton(Button.B)) {
-      climb.runWinch(-0.5);
+    if (manipulator.getAxis(Axis.RIGHT_TRIGGER) >= 0.1) {
+      climb.runWinch(0.5 * manipulator.getAxis(Axis.RIGHT_TRIGGER));
     } else {
-      climb.runWinch(0);
+      climb.runWinch(-0.5 * manipulator.getAxis(Axis.LEFT_TRIGGER));
```

Comment:
Why do we need to run the winch in reverse?

Reply:
In case there's not enough slack. Mostly for testing